tests/e2e/utils.go

```diff
@@ -114,3 +115,27 @@ func fillEtcdWithData(ctx context.Context, c *clientv3.Client, dbSize int) error
 	}
 	return g.Wait()
 }
+
+func getMemberIdByName(ctx context.Context, c *Etcdctl, name string) (id uint64, found bool, err error) {
+	resp, err := c.MemberList()
+	if err != nil {
+		return 0, false, err
+	}
+	for _, member := range resp.Members {
+		if name == member.Name {
+			return member.ID, true, nil
+		}
+	}
+	return 0, false, nil
+}
+
+func patchArgs(args []string, flag, newValue string) []string {
+	for i, arg := range args {
+		if strings.Contains(arg, flag) {
+			args[i] = fmt.Sprintf("--%s=%s", flag, newValue)
+			return args
+		}
+	}
+	args = append(args, fmt.Sprintf("--%s=%s", flag, newValue))
+	return args
+}
```

Comment:
Why the implementation is different from the main branch?

https://github.com/etcd-io/etcd/blob/93530f6e0aeaebc0f00c98b8688579a0e576e71b/tests/e2e/utils.go#L139-L147

Reply:
3.5 does not have "initial-cluster-state" as a default argument in e2e framework. So, we need to append it in `args`.

https://github.com/etcd-io/etcd/blob/93530f6e0aeaebc0f00c98b8688579a0e576e71b/tests/framework/e2e/cluster.go#L468-L473

First introduced in commit https://github.com/etcd-io/etcd/commit/6f63f4b676b775e7ce32bc9281d28c96f7165d9c

Reply:
We should backport https://github.com/etcd-io/etcd/pull/16707 to 3.5 if possible.

Please add a comment to explain the difference for now.